config/index.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     AUTH_URL: process.env.API_AUTH_URL,
     CLIENT_ID: process.env.API_CLIENT_ID,
     SECRET: process.env.API_SECRET,
+    TIMEOUT: 30000, // in seconds
```

Comment:
30000 seconds is 500 minutes. I think we can safely make this a bit shorter ;)

Reply:
Doh!

Reply:
@slorek sorry, it was the correct value, but wrong comment! It's actually milliseconds, not seconds!